packages/cypress-plugins/src/plugins/networkShim/clearNetworkFixtures.js

```diff
@@ -5,7 +5,7 @@ const rimraf = require('rimraf')
 
 module.exports = function clearNetworkFixtures({ fixturesFolder, env }) {
     try {
-        const serverMinorVersion = env.dhis2_server_minor_version.toString()
+        const serverMinorVersion = env.dhis2_api_version.toString()
```

Comment:
This could be made a constant or symbol as well

Reply:
Agreed, we can refactor this later though as it is an internal detail. Leaving it for now.